cmd/fleetctl/convert_test.go

```diff
@@ -11,7 +11,6 @@ import (
 )
 
 func TestConvertFileOutput(t *testing.T) {
-	t.Parallel()
```

Comment:
Perhaps counter-intuitively, this seemed to make tests slower when I tried them - maybe due to lock contention on stdout.